package/setup.py

```diff
@@ -286,27 +314,27 @@ def detect_openmp():
                     'MDAnalysis.visualization'],
           package_dir={'MDAnalysis': 'MDAnalysis'},
           ext_package='MDAnalysis',
-          ext_modules=extensions,
+          ext_modules=extensions(config),
           classifiers=CLASSIFIERS,
           long_description=LONG_DESCRIPTION,
           cmdclass=cmdclass,
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
           install_requires=[
-              'numpy>=1.5.0',  # currently not useful because without numpy we don't get here
```

Comment:
I also found a setuptools field that will install numpy prior to any setup or extension compilation that we are doing. I'll have a look if I can use that to have numpy installed automatically during the installation and thereby getting rid of the error message.